lib/nimble_template/addons/variants/phoenix/api/config.ex

```diff
@@ -17,12 +17,8 @@ defmodule NimbleTemplate.Addons.Phoenix.Api.Config do
   def edit_config_prod(%Project{otp_app: otp_app, web_module: web_module} = project) do
     Generator.replace_content(
       "config/prod.exs",
-      """
-      config :#{otp_app}, #{web_module}.Endpoint,
-      """,
-      """
-      config :#{otp_app}, #{web_module}.Endpoint.Anchor
-      """
+      "config :#{otp_app}, #{web_module}.Endpoint,",
+      "config :#{otp_app}, #{web_module}.Endpoint.Anchor"
```

Comment:
Hi @andyduong1920, I'm just a bit curious why we're using this approach to replace the content with `.Anchor` only to delete it in line 24. Is there a reason? 💭

Reply:
I have refactored it in https://github.com/nimblehq/elixir-templates/pull/188/commits/5e1006717fcffa9b6e4b0c4acce7049c9b8092f3 @bterone, please check the new version :)